frontend/src/components/UserAuth.js

```diff
@@ -67,27 +98,42 @@ const UserAuth = ({
       >
         <Box className="left-button-wrapper">
           <Button
-            className="font-inter bg-sky-500 hover:bg-sky-700 text-white font-bold rounded-2xl"
-            variant={'outlined'}
+            sx={{ px: 2 }}
+            className="font-inter bg-sky-500 hover:bg-sky-700 text-white font-bold rounded-xl"
             onClick={() => navigate(toggleDestination)}
           >
             {toggleText}
           </Button>
         </Box>
         <Box>
           <Button
-            className="font-inter bg-sky-500 hover:bg-sky-700 text-white font-bold rounded-2xl"
-            variant={'outlined'}
-            onClick={() => handleAuth(username, password)}
+            sx={{ px: 2 }}
+            className="font-inter bg-sky-500 hover:bg-sky-700 text-white font-bold rounded-xl"
+            onClick={() => {
```

Comment:
This part should be refactored, not very scalable especially if our form has more fields. You could potentially do it this way:

```js
onClick={() => {
  setUsernameSubmittedEmpty(username == '')
  setPasswordSubmittedEmpty(password == '')
  if (username && password) {
    handleAuth(username, password)
}
```

Reply:
Ah, I didn't think of that. Yeah, the code looks so much neater this way. I have changed it! :)